### PR TITLE
Feature/apm 318151 log forwarders add truncated to log msgs that are being truncated

### PR DESCRIPTION
**Pull request by IrynaKudlaienko**

### CODE REVIEW

**Review by bmrozinski-dt — APPROVED** [review]

File: tests/unit/test_content_trimmed.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    # given
+    log_entry = create_log_entry(log_message)
+    logs_ingest.main.content_length_limit = content_length_limit
+
+    # when
+    try:
+        actual_output = parse_record(log_entry, SelfMonitoring(execution_time=datetime.utcnow()))
+    finally:
+        # restore original value
+        logs_ingest.main.content_length_limit = content_length_limit_backup
```
Comment (wojtasskorcz) [inline tests/unit/test_content_trimmed.py:44]: Good approach doing it in `finally`!

File: tests/unit/test_content_trimmed.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    # then
+    expected_content = "{\"content\": \"WALTHAM, Mass.--(BUSINESS WIRE)-- Software intelligence company Dynatrace (N[TRUNCATED]"
```
Comment (wojtasskorcz) [inline tests/unit/test_content_trimmed.py:47]: One additional test could be to check if we're not overly eager in truncating stuff. For example, check if we're not truncating `"WALTHAM, Mass.--(BUSINESS WIRE)-- Software intelligence company Dynatrace (NYSE: DT)"`. From my experience, some software/webpages have these problems and it's super annoying. Not super important here though, just occurred to me

**Review by wojtasskorcz — APPROVED** [review]

File: tests/unit/test_content_trimmed.py (hunk 2 above)
Comment (IrynaKudlaienko) [reply tests/unit/test_content_trimmed.py:46]: Done

File: tests/unit/test_content_trimmed.py [hunk 3]
```diff
@@ -47,3 +46,27 @@ def test_content_trimmed():
...
     assert actual_output["content"] == expected_content
+
+
+def test_content_with_exact_len_not_trimmed():
+    message = "WALTHAM, Mass.--(BUSINESS WIRE)-- Software intelligence company Dynatrace (NYSE: DT)"
+    content_length_limit_backup = logs_ingest.main.content_length_limit
+
+    # given
+    log_entry = create_log_entry(message)
+    logs_ingest.main.content_length_limit = len(json.dumps(log_entry))
```
Comment (wojtasskorcz) [inline tests/unit/test_content_trimmed.py:57]: I would do it like in previous test:
```
content_length_limit = 100
```
or
```
content_length_limit = len(message)
```
Right now we're also counting the JSON braces and field name into the length limit so it wouldn't be truncated whether we were overly eager or not.

Comment (wojtasskorcz) [reply tests/unit/test_content_trimmed.py:57]: Nevermind, Ira explained to me how it works here

**Review by wojtasskorcz — APPROVED** [review]

**Pull request merged**